app/src/main/res/values/styles.xml

```diff
@@ -327,4 +327,39 @@
     <item name="android:textAllCaps">false</item>
     <item name="android:textColor">@color/oppiaPrimaryText</item>
   </style>
+  <!-- Start Over Lesson Button -->
+  <style name="StartOverLessonButton" parent="TextAppearance.AppCompat.Widget.Button">
+    <item name="android:layout_width">144dp</item>
+    <item name="android:layout_height">wrap_content</item>
+    <item name="android:paddingRight">12dp</item>
+    <item name="android:paddingLeft">12dp</item>
+    <item name="android:paddingTop">10dp</item>
+    <item name="android:paddingBottom">10dp</item>
+    <item name="android:background">@drawable/start_over_button_background</item>
+    <item name="android:fontFamily">sans-serif-medium</item>
+    <item name="android:drawableStart">@drawable/ic_start_over_24dp</item>
+    <item name="android:drawablePadding">4dp</item>
+    <item name="drawableTint">@color/oppiaPrimaryDark</item>
+    <item name="android:textAllCaps">true</item>
+    <item name="android:textColor">@color/oppiaPrimaryDark</item>
+    <item name="android:textSize">14sp</item>
+  </style>
+  <!-- Continue Lesson Button -->
+  <style name="ContinueLessonButton" parent="TextAppearance.AppCompat.Widget.Button">
+    <item name="android:layout_width">144dp</item>
+    <item name="android:layout_height">wrap_content</item>
+    <item name="android:drawablePadding">4dp</item>
+    <item name="android:paddingRight">12dp</item>
+    <item name="android:paddingLeft">12dp</item>
+    <item name="android:paddingTop">10dp</item>
+    <item name="android:paddingBottom">10dp</item>
```

Comment:
Do not mention paddingTop and paddingBottom. Instead mention `minHeight = 48dp` as we need this to make this correctly accessible

Reply:
done